packages/server/src/__tests__/standalone/integration.test.ts

```diff
@@ -0,0 +1,28 @@
+import { httpServer as getHttpServer } from '../../standalone';
+import type { BaseContext } from '../../externalTypes';
+import type {
+  CreateServerForIntegrationTestsOptions,
+  CreateServerForIntegrationTestsResult,
+} from '../integration';
+import { defineIntegrationTestSuite } from '../integration';
+import type { ApolloServerOptions } from '../../types';
+import { ApolloServer } from '../../ApolloServer';
+
+defineIntegrationTestSuite(async function (
+  serverOptions: ApolloServerOptions<BaseContext>,
+  testOptions?: CreateServerForIntegrationTestsOptions,
+): Promise<CreateServerForIntegrationTestsResult> {
+
+  // This test option doesn't apply to the standalone version, this replicates
+  // the behavior expected by the integration test suite.
```

Comment:
we were maybe gonna move this to express-specific tests?

Reply:
[f6b44fd](https://github.com/apollographql/apollo-server/pull/6420/commits/f6b44fd4e533e3abad21f0bd4029b816888a9bf3)